cams:
  CAM_BACK:
    cam_intrinsic:
      - [809.2209905677063, 0.0, 829.2196003259838]
      - [0.0, 809.2209905677063, 481.77842384512485]
      - [0.0, 0.0, 1.0]
    data_path: ./data/nuscenes/samples/CAM_BACK/n015-2018-08-02-17-16-37+0800__CAM_BACK__1533201470437525.jpg
    sensor2ego_rotation:
      [0.5037872666382278, -0.49740249788611096, -0.4941850223835201, 0.5045496097725578]
    sensor2ego_translation: [0.0283260309358, 0.00345136761476, 1.57910346144]
    sensor2lidar_rotation:
      - [-0.9999413173368413, 0.009846645553456103, -0.004517239644395279]
      - [0.004591280151619832, 0.0075097729192299114, -0.9999612609782784]
      - [-0.009812340660088988, -0.9999233205011492, -0.007554540934044881]
    sensor2lidar_translation: [-0.0036954598058400734, -0.9075747504218441, -0.283221870904093]
  CAM_BACK_LEFT:
    cam_intrinsic:
      - [1256.7414812095406, 0.0, 792.1125740759628]
      - [0.0, 1256.7414812095406, 492.7757465151356]
      - [0.0, 0.0, 1.0]
    data_path: ./data/nuscenes/samples/CAM_BACK_LEFT/n015-2018-08-02-17-16-37+0800__CAM_BACK_LEFT__1533201470447423.jpg
    sensor2ego_rotation:
      [0.6924185592174665, -0.7031619420114925, -0.11648342771943819, 0.11203317912370753]
    sensor2ego_translation: [1.03569100218, 0.484795032713, 1.59097014818]
    sensor2lidar_rotation:
      - [-0.3170637040675469, 0.019895609309549686, -0.9481955348413992]
      - [0.9480779188624351, 0.03287371923632269, -0.3163345987226865]
      - [0.024877044206231273, -0.9992614689428244, -0.029285651056243037]
    sensor2lidar_translation: [-0.48313021193087025, 0.09925074956760227, -0.24976867779076528]
  CAM_BACK_RIGHT:
    cam_intrinsic:
      - [1259.5137405846733, 0.0, 807.2529053838625]
      - [0.0, 1259.5137405846733, 501.19579884916527]
      - [0.0, 0.0, 1.0]
    data_path: ./data/nuscenes/samples/CAM_BACK_RIGHT/n015-2018-08-02-17-16-37+0800__CAM_BACK_RIGHT__1533201470427893.jpg
    sensor2ego_rotation:
      [0.12280980120078765, -0.132400842670559, -0.7004305821388234, 0.690496031265798]
    sensor2ego_translation: [1.0148780988, -0.480568219723, 1.56239545128]
    sensor2lidar_rotation:
      - [-0.3567246414755177, -0.005413898071006906, 0.934193887729864]
      - [-0.9335307099588551, 0.04018099149456167, -0.3562385457614421]
      - [-0.035608197481429044, -0.9991777507681946, -0.01938758989490032]
    sensor2lidar_translation: [0.482312676691663, 0.0791837770804591, -0.27307179836459383]
  CAM_FRONT:
    cam_intrinsic:
      - [1266.417203046554, 0.0, 816.2670197447984]
      - [0.0, 1266.417203046554, 491.50706579294757]
      - [0.0, 0.0, 1.0]
    data_path: ./data/nuscenes/samples/CAM_FRONT/n015-2018-08-02-17-16-37+0800__CAM_FRONT__1533201470412460.jpg
    sensor2ego_rotation:
      [0.4998015430569128, -0.5030316162024876, 0.4997798114386805, -0.49737083824542755]
    sensor2ego_translation: [1.70079118954, 0.0159456324149, 1.51095763913]
    sensor2lidar_rotation:
      - [0.9999693728739966, 0.006755601474249302, -0.003951602548949129]
      - [0.003824557675651031, 0.0187164545279605, 0.9998175168942008]
      - [0.006828328680530752, -0.9998020084889915, 0.018690044109277416]
    sensor2lidar_translation: [-0.012715806721303125, 0.7688055822719093, -0.31059455973338856]
  CAM_FRONT_LEFT:
    cam_intrinsic:
      - [1272.5979470598488, 0.0, 826.6154927353808]
      - [0.0, 1272.5979470598488, 479.75165386361925]
      - [0.0, 0.0, 1.0]
    data_path: ./data/nuscenes/samples/CAM_FRONT_LEFT/n015-2018-08-02-17-16-37+0800__CAM_FRONT_LEFT__1533201470404874.jpg
    sensor2ego_rotation:
      [0.6757265034669446, -0.6736266522251881, 0.21214015046209478, -0.21122827103904068]
    sensor2ego_translation: [1.52387798135, 0.494631336551, 1.50932822144]
    sensor2lidar_rotation:
      - [0.5726028043564848, 0.0027092489875872503, -0.8198284505998853]
      - [0.8195566437415105, 0.024067561593338945, 0.5724924979229841]
      - [0.021282296451183572, -0.9997066632011964, 0.011560770255041255]
    sensor2lidar_translation: [-0.4917212028847189, 0.5936531098472528, -0.31925386662383204]
  CAM_FRONT_RIGHT:
    cam_intrinsic:
      - [1260.8474446004698, 0.0, 807.968244525554]
      - [0.0, 1260.8474446004698, 495.3344268742088]
      - [0.0, 0.0, 1.0]
    data_path: ./data/nuscenes/samples/CAM_FRONT_RIGHT/n015-2018-08-02-17-16-37+0800__CAM_FRONT_RIGHT__1533201470420339.jpg
    sensor2ego_rotation:
      [0.2060347966337182, -0.2026940577919598, 0.6824507824531167, -0.6713610884174485]
    sensor2ego_translation: [1.5508477543, -0.493404796419, 1.49574800619]
    sensor2lidar_rotation:
      - [0.5518728006969844, -0.010452325915577761, 0.8338627949092215]
      - [-0.8335213795734647, 0.024319671348433384, 0.5519516857293306]
      - [-0.0260484480307739, -0.9996495898405882, 0.004709128022186309]
    sensor2lidar_translation: [0.49650027090206095, 0.6174621492795893, -0.32655958795607276]
ego2global_rotation:
  [0.9984303573176436, -0.008635865272570774, 0.0025833156025800875, -0.05527720957189669]
ego2global_translation: [249.89610931430778, 917.5522573162784, 0.0]
lidar2ego_rotation:
  [0.7077955119163518, -0.006492242056004365, 0.010646214713995808, -0.7063073142877817]
lidar2ego_translation: [0.943713, 0.0, 1.84023]
scene_token: e7ef871f77f44331aefdebc24ec034b7
timestamp: 1533201470448696
